default/stringtables/en.txt

```diff
@@ -124,6 +124,9 @@ Back
 LAST
 Last
 
+DIFFICULTY
+Difficulty
+
```

Comment:
I'd avoid newbie, although I've never understood why I've seen it interpreted as an insult too many times, I'm guessing there are groups /cultures out there that use it as such that I've managed to avoid. New Player would be OK but Beginner is clearer. 

We had a suggestion to have a catch all difficulty recommended settings option to set various things to "easier" options, I'm not sure it's needed but I don't think it matters. All the AI types except Beginner can be a challenge but the meaning of that specific setting is clear. 

Not 100% sure this is the approach to take but something's needed and we can improve it or replace it in future.